### ISSUE TITLE
Items disappear if I left items in crafting table

### ISSUE THREAD

**Issue by XyLyXyRR** [kyrptonaught/quickshulker#38]
Right click crafting table, put some items in it, and close inventory will cause items disappear.

**Comment by kyrptonaught** [comment]
Hmm got any more details? The items are returning to my inventory just fine

**Comment by XyLyXyRR** [comment]
Open inventory, right click crafting table, put a few items in there, close inventory, and these items is gone.

**Comment by XyLyXyRR** [comment]
And I can't craft anything in this way.

**Comment by kyrptonaught** [comment]
Are you playing on a server? Are you using the latest version 1.2.3?

**Comment by XyLyXyRR** [comment]
No, I'm using it on single player world, and is mc 1.16.5, quick shulker 1.1.14.

**Comment by kyrptonaught** [comment]
Ah it's 1.16, I was checking 1.17
